app/services/katello/pulp3/content_view_version/export.rb

```diff
@@ -3,15 +3,16 @@ module Pulp3
     module ContentViewVersion
       class Export
         include ImportExportCommon
+        attr_accessor :smart_proxy, :content_view_version, :destination_server, :from_content_view_version
```

Comment:
Sorry if I missed it... Do these need to be accessors ie someone needs to be able to set these values on an instance of this class from the outside?

Reply:
Maybe attr_reader would be more appropriate if accessor is not needed

Reply:
I could do the reader, that sounds better. I just need them to be accessible to the metadata generator. The generator is using methods from the service class to fetch repository info.

Reply:
updated